compat/mingw.c

```diff
@@ -333,7 +333,7 @@ static const wchar_t *make_relative_to(const wchar_t *path,
 
 	len = wcslen(path);
 	if (i + len + 1 > size) {
-		error("Could not make '%S' relative to '%S' (too large)",
+		error("Could not make '%ls' relative to '%ls' (too large)",
```

Comment:
I'll leave this one here for your series since it doesn't have anything to do with FSMonitor (and my series is already big enough).